stdlib/regex/src/test/java/org/ballerinalang/stdlib/regex/RegexTest.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.ballerinalang.stdlib.regex;
+
+import org.ballerinalang.model.values.BString;
+import org.ballerinalang.model.values.BValue;
+import org.ballerinalang.model.values.BValueArray;
+import org.ballerinalang.test.util.BCompileUtil;
+import org.ballerinalang.test.util.BRunUtil;
+import org.ballerinalang.test.util.CompileResult;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+/**
+ * Class to test functionality of regex.
+ *
+ * @since 1.0
+ */
+public class RegexTest {
+
+    private CompileResult result = BCompileUtil.compile("test-src/regex.bal");
```

Comment:
Shall we use common path separator here instead of `/` ? Otherwise windows tests will fail due to this. You can refer [this test](https://github.com/ballerina-platform/ballerina-lang/blob/ad3aa5526df20d8171abd47bb5eb8c57a41b2f61/stdlib/task/src/test/java/org/ballerinalang/stdlib/task/service/AppointmentServiceTest.java#L45) for an example.

Reply:
Done